server/views/pages/categoriserHome.html

```diff
@@ -76,7 +76,7 @@ <h1 class="govuk-heading-l">Prisoner Categorisation</h1>
       <td class="govuk-table__cell">
         {% if row.displayStatus == 'Back from Supervisor' %}
           {{ govukTag({
-            html: 'Rejected by<br/>supervisor'
+            html: 'REJECTED BY<br/>SUPERVISOR'
```

Comment:
Is this supposed to have been capitalised?

Reply:
Yes, it looks (from the tests) that there must have used to be css which capitalised this, and that must have been removed with this upgrade, since the test expects it to be in all caps I've capitalised it

Reply:
👍